CMSgov/onemac
# Oy2 20975 - add temp ext type to details page

Story: https://qmacbis.atlassian.net/browse/OY2-20975
Endpoint: See github-actions bot comment

### Details

Show the temporary extension type on the details page for temporary extensions.

### Changes

- Made the temporaryExtensionTypes an exportable const array so that it can be reused
- Updated details page to lookup the type label from temporaryExtensionType using the value


### Test Plan

1. Login as a state submitter
2. Submit a temporary extension and note the chosen Temporary Extension Type
3. Navigate to the details page of the TE that was submitted
4. Verify the Temporary Extension Type field exists and reflects the chosen type from step 2
5. Verify the page title reflects the ACs
6. Verify the date submitted field label reflects the ACs
